src/components/charts/MultiChartSummary.tsx

```diff
@@ -30,33 +31,48 @@ const MultiChartSummary: React.FC<MultiChartSummaryProps> = ({
     name,
     setChartDetailsUpdate,
 }) => {
+
+    const removeAllCharts = () => {
+        removeChart(-1)
+    }
+
     return (
         <div className="chart-group--summary">
             {chartListing && (
                 <>
-                    {name && <div className="flex column left border-bottom mb-20">
-                        <span className="flex flex-justify w-100 fs-14 cn-9">
-                            Group name
-                            <Pencil className="pointer" onClick={(e) => setChartDetailsUpdate(true)} />
-                        </span>
-                        <div className="flex left fw-6 fs-14 mt-8 mb-20">{name}</div>
-                    </div>}
-                    <div
-                        className={`selected-chart-widget p-12 select-chart cursor ${
-                            typeof configureChartIndex !== 'number' ? 'active' : ''
-                        }`}
-                        style={{ gridTemplateColumns: '40px 1fr 24px' }}
-                        onClick={chartListing}
-                    >
-                        <Forward className="rotate anchor" style={{ ['--rotateBy' as any]: '180deg' }} />
-                        <div className="anchor ml-18">Select Charts</div>
-                        <span />
-                    </div>
-
+                    {name && (
+                        <div className="flex column left border-bottom mb-20">
+                            <span className="flex flex-justify w-100 fs-14 cn-9">
+                                Group name
+                                <Pencil className="pointer" onClick={(e) => setChartDetailsUpdate(true)} />
```

Comment:
We create separate function for {(e) => setChartDetailsUpdate(true)} and avoid inline function

Reply:
done